LeapEngine/GameContext/Window.h

```diff
@@ -19,6 +21,19 @@ namespace leap
 		const glm::ivec2& GetWindowSize() const;
 		bool IsFullScreen() const { return m_FullScreen; }
 		void SetFullScreen(bool value);
+		void SetResize(bool value) const;
+		void SetIcon(const std::string& path) const;
+		void SetWindowTitle(const std::string& text) const;
+		void SetMinimized(bool value) const;
+		void SetHiden(bool value) const;
+		void RequireAttention() const;
+		void SetPosition(const glm::ivec2& position) const;
+		void SetAspectRatio(const glm::ivec2& size) const;
+
+		// Warning this function overrides everything the SetLimitSize does.
+		void SetMinimumSize(const glm::ivec2& size) const;
```

Comment:
Add overloads that take two integers (same for other functions that take glm arguments)